src/app/admin/user-management/user-management.styl

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ */
+$title-color = #dfdfdf
```

Comment:
this color definition should not be in a child stylus file, it's harder to track